bagh/bagh.json

```diff
@@ -5,6 +5,14 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "versioning": {
+    "request_parameter": "versie",
```

Comment:
Ik zou willen voorstellen dat dit altijd hetzelfde heet als wat je hieronder het 'version_field_name' noemt. Dan kan dit dus weg en dan heet de request parameter in dit geval 'volgnummer'.